suites/pacific/rgw/tier-2_rgw_test_5-3_specific_fixes.yaml

```diff
@@ -187,6 +187,7 @@ tests:
       desc: Test rgw multipart upload through curl
       polarion-id: CEPH-9801
       module: sanity_rgw.py
+      comments: known issue (BZ2266793) targeted to 6.1z7
```

Comment:
remove all timeout from this suite of pacific

Reply:
@anrao19 
I removed timeout from only my tests in the PR
there are many such places where timeout is present in all of our RGW suites
we can take it in another PR know
why stressing in this PR

Reply:
done, pls review